include/prism/options.h

```diff
@@ -85,8 +85,13 @@ typedef struct {
     /** A bitset of the various options that were set on the command line. */
     uint8_t command_line;
 
-    /** Whether or not the frozen string literal option has been set. */
-    bool frozen_string_literal;
+    /**
+    * Whether or not the frozen string literal option has been set.
+    *   -1 is explicitly disabled.
+    *    0 is not set (default).
+    *    1 is explicitly enabled.
+    */
+    int8_t frozen_string_literal;
```

Comment:
Can you make some constants so we don't rely on the specific values everywhere? Something like:

```c
static const int8_t PM_OPTIONS_FROZEN_STRING_LITERAL_DISABLED = -1;
static const int8_t PM_OPTIONS_FROZEN_STRING_LITERAL_UNSET = 0;
static const int8_t PM_OPTIONS_FROZEN_STRING_LITERAL_ENABLED = 1;
```

Reply:
I made them `#define` otherwise they can't be used in `switch/case`.